native/build.go

```diff
@@ -47,16 +45,27 @@ type Build struct {
 func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 	b.Logger.Title(context.Buildpack)
 	result := libcnb.NewBuildResult()
-
-	manifest, err := libjvm.NewManifest(context.Application.Path)
-	if err != nil {
-		return libcnb.BuildResult{}, fmt.Errorf("unable to read manifest in %s\n%w", context.Application.Path, err)
-	}
+	var manifest *properties.Properties
+	var startClass string
+	var buildFromJar bool
```

Comment:
This is just Go-style note. It's not common to see variables declared at the beginning of a method like this. Typically you use the smallest scope possible for the variable, declaring it with `:=`. It seems like that would be possible here, although I didn't pop open VS code and try it.

My preference would be to use `:=` and declare them as needed or define the `var` at the point it's needed as opposed to the top of this file.

Reply:
I just did a little refactor for this code, just let me know